pom.xml

```
@@ -196,7 +196,7 @@
             <dependency>
                 <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-servlets</artifactId>
-                <version>${jetty.servlets.version}</version>
+                <version>${jetty.version</version>
```

@sumit-confluent please fix the missing bracket 
```suggestion
                <version>${jetty.version}</version>
```

---

thanks for pointing out!! added the missing bracket